pkg/operator/operator.go

```diff
@@ -370,6 +363,23 @@ func (o *Operator) ensureDefaultIngressController(infraConfig *configv1.Infrastr
 			Replicas: &replicas,
 		},
 	}
+	if ingressConfig.Spec.LoadBalancer.Platform.Type == configv1.AWSPlatformType {
```

Comment:
`ingressConfig` cannot be nil, but it might be uninitialized.  That's a good point—we should return instead of calling `o.ensureDefaultIngressController` here.